include/vm/async.h

```diff
@@ -34,7 +34,6 @@ template <typename T> class Async {
         std::thread([FPtr, P = std::move(Promise),
                      Tuple = std::tuple(
                          &TargetVM, std::forward<ArgsT>(Args)...)]() mutable {
-          std::get<0>(Tuple)->newThread();
```

Comment:
Do we need to call prepare here?
It looks like the `newThread` is replaced by `prepare`.

Reply:
No more `prepare` required here.